app/views/episodes/_form.html.erb

```diff
@@ -15,10 +15,7 @@
       </div>
     </div>
     <div class="col-lg-4 d-grid align-content-start gap-3">
-      <div class="sticky-lg-top">
-        <%= render "form_status", episode: episode, form: form %>
-        <%= render "form_distribution", episode: episode, form: form %>
-      </div>
+      <%= render "form_status", episode: episode, form: form %>
```

Comment:
Basically, I moved all the wrappers for the status into the status file, so I could keep the modal out of the div.